package.json

```diff
@@ -29,6 +29,8 @@
   ],
   "devDependencies": {
     "babel-preset-es2015": "^6.18.0",
+    "eslint-config-incuna": "1.0.0",
+    "eslint-plugin-import": "2.2.0",
```

Comment:
I think projects could do with `^` to allow up-to-next-major-version upgrades, so shared lint config changes can be more easily propagated to projects.